schema/app.json

```diff
@@ -116,33 +120,39 @@
                 "png": {
                     "type": "string",
                     "format": "uri-reference",
-                    "description": "URI to the PNG format of the application logo"
+                    "description": "URI to the PNG format of the application logo."
                 },
                 "svg": {
                     "type": "string",
                     "format": "uri-reference",
-                    "description": "URI to the SVG format of the application logo"
+                    "description": "URI to the SVG format of the application logo."
                 }
             }
         },
         "serviceURL": {
             "type": "object",
+            "description": "For mainnet and testnet, the deployments need to be secured with the requirement of https, wss and publicKey. Other networks can have insecure deployments but in that case http and websocket endpoints are mandatory.",
             "required": [
-                "http",
-                "ws"
+                "ws",
+                "http"
             ],
             "properties": {
                 "ws": {
                     "type": "string",
                     "format": "uri-reference",
-                    "description": "WS endpoint to access Lisk Service API against the blockchain application"
+                    "description": "WS(S) endpoint to access Lisk Service API against the blockchain application. In case of mainnet and testnet deployments WSS url is required."
                 },
                 "http": {
                     "type": "string",
                     "format": "uri-reference",
-                    "description": "HTTP endpoint to access Lisk Service API against the blockchain application"
+                    "description": "HTTP(S) endpoint to access Lisk Service API against the blockchain application. In case of mainnet and testnet deployments HTTPS url is required."
```

Comment:
```suggestion
                    "description": "HTTP(S) endpoint to access Lisk Service API against the blockchain application. In case of mainnet and testnet deployments, a HTTPS URL is required."
```